### ISSUE TITLE
[A11y] Dashboard - Landmark Roles

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1845]
### Operating System

Windows

### Browser

Edge

### Affected page

Backend

### Admin user?

Yes

### Issue Description

The "main" landmark wraps a huge set of links which are repeated throughout the admin dashboard.

The "navigation" landmark around the admin links is called "sidebar-nav", which doesn't usefully describe the meaning.

### Suggested Fix

The "main" landmark needs to only wrap the content which isn't repeated and isn't navigation-related content.

The sidebar should be given a more human-friendly name.  Even having no name at all could arguably be better as it's about as meaningful but less verbose.

**Comment by jechols** [comment]
QA: inspect elements.  Tools which add an overlay to show what is wrapped in a given region are notoriously inaccurate with floating elements, so I'd strongly advise looking at the actual DOM.
